lib/ParamedicApp.js

```diff
@@ -39,6 +39,12 @@ class ParamedicApp {
         this.isBrowser = this.platformId === utilities.BROWSER;
         this.isIos = this.platformId === utilities.IOS;
         this.isWindows = this.platformId === utilities.WINDOWS;
+
+        logger.info('---------------------------------------------------------');
```

Comment:
The mismatch of logger method used here somehow happened via a borked cherry pick merge, will be fixed via https://github.com/apache/cordova-paramedic/pull/171